### PR TITLE
Fix for ilovepdf.com

### PR DESCRIPTION
**Pull request by rakleed**
- Remove the share buttons on the page with the converted file.

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -36,6 +36,7 @@ sonsaat.com.tr,okudunmuhaberi.com,spooor.com,mavididim.com.tr,filmfiz.net,filmw.
...
 !
 focus.ua##.c-post__social
+ilovepdf.com##.card
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:39]: `.card` isn't as unique as `.download__add`, so it's better to replace it to `ilovepdf.com##.download__add`.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply SocialFilter/sections/specific.txt]: Nvm, I made changes.

Comment (Rakleed) [reply SocialFilter/sections/specific.txt]: @zloyden I checked again, and for me the `.download__add` rule does not work, but the `.card` rule works.

Comment (zloyden) [reply SocialFilter/sections/specific.txt]: It works fine on my end. Can you please check `ilovepdf.com#?#.container > div[class]:has(> div.card div.share)` in the User rules?

Comment (Rakleed) [reply SocialFilter/sections/specific.txt]: The rule `ilovepdf.com#?#.container > div[class]:has(> div.card div.share)` works for me.

Comment (zloyden) [reply SocialFilter/sections/specific.txt]: Fine, it was added recently. Filters update should be available in 1-2 hour.

Comment (Rakleed) [reply SocialFilter/sections/specific.txt]: Why does the `ilovepdf.com##.download__add` rule work only in the extension, but not in the application? Is this an app bug or platform limitation?

Comment (zloyden) [reply SocialFilter/sections/specific.txt]: The extension and the app have a different approach to blocking elements in such cases, but we have a possibilites to block it via app. Thanks for your feedback:)

**Pull request merged**